packages/utils/src/bn.ts

```diff
@@ -0,0 +1,11 @@
+import BN from 'bn.js'
+
+export function compareBN(a: BN, b: BN) {
```

Comment:
We generally use BigNumber instead of BN.js whenever we can

Reply:
Yeah, I haven't brought up that discussion yet.
web3 defaults to BN; to use BigNumber we need to purposely set it with each new contract instance. Which if we don't do can result in really confusing errors.

So i wanted to bring this up, and see if we want to review our library of choosing.

Reply:
+1 on having that conversation, but can we decouple it from this PR and use `BigNumber` for now?

Reply:
I'll leave this for now (the code is inherited from cli) and focus on other things.
It will be addressed in a later PR